### PR TITLE
Youngee progress

### PR DESCRIPTION
**Pull request by youngee91**

### CODE REVIEW

File: virtual-cluster/youngee.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+Use `grep -cw vmx /proc/cpuinfo`, a non-zero response confirms that nested virtualization is enabled.
+
+Use `ifconfig` to check the subnet for eth0. The subnet should be 255.255.240.0 if the multiple subnet is enabled successfully.
+
+Create VPC network, choose a your region and IP range
+
+## Installing Proxmox
+
+Reference to https://pve.proxmox.com/wiki/Install_Proxmox_VE_on_Debian_Stretch for installation of Proxmox on Debian Stretch.
```
Comment (eugene-chow) [inline virtual-cluster/youngee.md:50]: Is there a reason why you're installing PVE on Debian instead of using the ISO?

**Review by eugene-chow — APPROVED** [review]
Good work. Thanks!

File: virtual-cluster/youngee.md (hunk 1 above)
Comment (youngee91) [reply virtual-cluster/youngee.md:50]: The PVE ISO is not available on Google Cloud and uploading the ISO file might incur charges. Therefore, our approach is to create a available Debian image first and install the PVE on to it.

**Comment by eugene-chow** [comment]
I see. Take note that only the ISO version comes with commercial support. You guys might need it someday.

The problem with the ISO is that it makes unattended installs difficult or impossible. Only the Debian + PVE method supports unattended installation.

**Pull request merged**